### PR TITLE
Account holder view

### PR DESCRIPTION
**Pull request by slkendell-mt**
https://trello.com/c/7fdu17PC/721-account-holder-view

### CODE REVIEW

File: backoffice/src/views/AccountHolderView.tsx [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+  // };
+
+  return (
+    <div className={classes.root}>
+      <PageHeader>Account Holder: {accountHolder?.fullName}</PageHeader>
+      <PageContent>
+        <Card>
+          <CardContent>
+            <PanelViewingState
+              fields={[
```
Comment (slkendell-mt) [inline backoffice/src/views/AccountHolderView.tsx:112]: Need to tidy this up, currently inline field def

File: backoffice/src/views/AccountHolderView.tsx [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+                      target="_blank"
+                      href={`/backoffice#/beacons/${beacon.id}`}
+                      variant="outlined"
+                      sx={{ marginY: "auto" }}
+                    >
+                      View
+                    </Button>
+                  </Grid>
+                  <PanelViewingState
+                    fields={[
```
Comment (slkendell-mt) [inline backoffice/src/views/AccountHolderView.tsx:174]: Need to tidy this up, currently inline field def

**Review by SamBrownMT — APPROVED** [review]

File: backoffice/src/gateways/mappers/BeaconResponseMapper.ts [hunk 3]
```diff
@@ -77,6 +85,70 @@ export class BeaconResponseMapper implements IBeaconResponseMapper {
...
   }
 
+  public mapAccountHolder(
+    accountHolder: IAccountHolderResponse
+  ): IAccountHolder {
+    return {
+      id: accountHolder.id,
+      fullName: accountHolder.attributes.fullName || "",
+      email: accountHolder.attributes.email || "",
+      telephoneNumber: accountHolder.attributes.telephoneNumber || "",
```
Comment (Evie-Skinner18) [inline backoffice/src/gateways/mappers/BeaconResponseMapper.ts:95]: Consider moving mapping logic server side?

Comment (slkendell-mt) [reply backoffice/src/gateways/mappers/BeaconResponseMapper.ts:95]: Yeah fair point, it probably should be, but at the moment we're re-suing the beacon workflow and could cause issues

**Pull request merged**